jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -228,6 +228,25 @@ def test_multiprocessing2():
 @pytest.mark.parametrize("method", ['OLS'])  # don't do GLS to see if it causes hang
 class TestMethods:
 
+    def test_segment_weighting(self, method):
+        # all pixel values are zero. So slope should be zero
```

Comment:
this comment doesn't seem to match the actual data values being set below, which are not all zero

Reply:
Ended up removing this test, as I didn't see its utility. Added some other tests instead.